src/components/chartRepo/ChartRepo.tsx

```diff
@@ -114,7 +114,14 @@ export default function ChartRepo({ isSuperAdmin }: ChartRepoType) {
                 .sort((a, b) => a.name.localeCompare(b.name))
                 .map(
                     (chart) =>
-                        chart.id != 1 && <CollapsedList {...chart} allowInsecureConnection={chart.allow_insecure_connection} key={chart.id || getRandomInt()} reload={reload} />,
+                        chart.id != 1 && (
+                            <CollapsedList
+                                {...chart}
+                                allowInsecureConnection={chart.allow_insecure_connection}
+                                key={chart.id || getRandomInt()}
```

Comment:
this is incorrect 

for every render, this will be unmounted and remounted due to random integer as key whenever chart id is not present

Reply:
We can avoid for now, Got the changes from formatting